src/index.html

```diff
@@ -4,9 +4,402 @@
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class = "header">
+      <a href = "#main" class="header__logo-link">
+        <img
+          src = "images/logo.png"
+          alt = "logo"
+          class = "header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class = "nav___list">
+          <li class = "nav___item">
+            <a class = "nav___link is-active" href = "#apple">Apple</a>
+          </li>
+
+          <li class = "nav___item">
+            <a class = "nav___link" href = "#samsung">Samsung</a>
+          </li>
+
+          <li class = "nav___item">
+            <a class = "nav___link" href = "#smartphones">Smartphones</a>
+          </li>
+
+          <li class = "nav___item">
+            <a
+              class = "nav___link"
+              href = "#laptops_&_computers"
+              data-qa = "nav-hover">
+              Laptops&nbsp;&&nbsp;Computers
+            </a>
+          </li>
+
+          <li class = "nav___item">
+            <a class = "nav___link" href = "#gadgets">Gadgets</a>
+          </li>
+
+          <li class = "nav___item">
+            <a class = "nav___link" href = "#tablets">Tablets</a>
+          </li>
+
+          <li class = "nav___item">
+            <a class = "nav___link" href = "#photo">Photo</a>
+          </li>
+
+          <li class = "nav___item">
+            <a class = "nav___link" href = "#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class = "container">
+      <div class = "card" data-qa = "card">
+        <img
+          class = "card__product-pic"
+          src = "images/imac.jpeg"
+          alt = "APPLE A1419 iMac"
+        >
+
+        <p class="card__product-info">
+          APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__product-reviews">
+          <div class = "stars stars--4">
+            <div class = "stars__star"></div>
+            <div class = "stars__star"></div>
+            <div class = "stars__star"></div>
+            <div class = "stars__star"></div>
+            <div class = "stars__star"></div>
+          </div>
+
+          <div class="card__product-reviews-quantity">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__product-price">
+          <div class="card__product-price-word">Price:</div>
+          <div class="card__product-price-quantity">$2,199</div>
+        </div>
+
+        <a
+          class="card__button-buy"
+          href="#"
+          data-qa = "card-hover">
+            buy
+        </a>
```

Comment:
Fix code style everywhere
```suggestion
        <a
          class="card__button-buy"
          href="#"
          data-qa="card-hover"
        >
          buy
        </a>
```